site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -257,6 +257,32 @@ paths:
             403:
                $ref: '#/components/responses/NotAllowed'
 
+      get:
+         tags:
+            - devices
+         summary: Search devices for a tenant, with optional filters and paging options.
+         operationId: searchDevicesForTenant
+         requestBody:
+            description: Filters, paging and sorting options.
+            content:
+               application/json:
+                  schema:
+                     $ref: '#/components/schemas/PagingSortingFilteringOptions'
+            required: false
+         responses:
+            200:
+               description: operation successful
+               content:
+                  application/json:
+                     schema:
+                        $ref: '#/components/schemas/Device'
```

Comment:
this will most likely be an array of devices instead of just a single device, won't it?